File: Python/avgoflst.py
def avg_val(lst):
    lst_sum = 0
    for i in range(len(lst)):
        lst_sum += lst[i]
    return lst_sum / len(lst)


if __name__ == "__main__":
    l = list(map(int, input("Please enter list values: ").split()))
    avg = avg_val(l)
    print(f"Average value is: {avg}")

File: C++/avgoflst.cpp
#include <iostream>
using namespace std;

float avgOfLst(int lst[], int size);

int main() {
    int test[] = {19, 2, 20, 1, 0, 18};
    int avg = avgOfLst(test, 6);
    cout << avg << endl;
    return 0;
}


float avgOfLst(int lst[], int size) {
    int sum = 0;
    for (int i = 0; i < size; i++) {
        sum += lst[i];
    }
    return sum / size;
}
